recipes-bsp/u-boot/u-boot_%.bbappend

```diff
@@ -4,6 +4,8 @@ SRC_URI:append:rpi = " \
     file://fw_env.config \
 "
 
+SRC_URI:append = " file://0001-rpi-always-set-fdt_addr-with-firmware-provided-FDT-address.patch"
```

Comment:
I'm wondering if it would be a bit better to use the `rpi` override. The advantage I see is that other setups would not break if the patch doesn't apply anymore forcing only the Raspberry Pi machines to update the patch. Given that this is a Raspberry Pi specific patch, it would make sense.